default/python/AI/AIstate.py

```diff
@@ -467,14 +483,27 @@ def update_system_status(self):
             mob_rating = CombatRatingsAI.combine_ratings_list(mob_ratings)
             if fleetsLostBySystem.get(sys_id, []):
                 lost_fleet_rating = CombatRatingsAI.combine_ratings_list(fleetsLostBySystem[sys_id])
-            if not system or partial_vis_turn == -9999:  # under current visibility rules should not be possible to have any losses or other info here, but just in case...
+            # under current visibility rules should not be possible to have any losses or other info here,
+            # but just in case...
+            if not system or partial_vis_turn == -9999:
                 if verbose:
-                    print "Have never had partial vis for system %d ( %s ) -- basing threat assessment on old info and lost ships" % (sys_id, sys_status.get('name', "name unknown"))
+                    print ("Have never had partial vis for system %d ( %s ) "
+                           "-- basing threat assessment on old info and lost ships") % (
+                        sys_id, sys_status.get('name', "name unknown"))
```

Comment:
Not necessary in this PR but log statement could be replaced with the more concise
```print "Never had partial vis for %s - basing threat assessment on old info and lost ships" % system```